pages/padrinhos/cadastro-padrinho.js

```diff
@@ -0,0 +1,210 @@
+import { useState, useEffect } from 'react';
+import Layout from 'components/Layout/Layout';
+import { getData } from 'functions/getData';
+import FormField from 'components/Blocks/Form/components/FormField';
+import getRecords from 'functions/getRecords';
+
+// TODO: CLEAN CODE!!!
+
+const Padrinhos = (props) => {
+  const [formData, setFormData] = useState({ ...headers });
+  const [status, setStatus] = useState();
+  const [isLoading, setIsLoading] = useState(false);
+  const [imageSrc, setImageSrc] = useState();
+
+  // CHANGE
+  // ===================================
+  const handleOnChange = (e) => {
+    if (e.target.files) {
+      const reader = new FileReader();
+      reader.onload = function (onLoadEvent) {
```

Comment:
Please create a function to validate the date or change the type to date. If there's no field type associated to it, please create one.

Reply:
Done